dist/index.js

```diff
@@ -65301,6 +65301,16 @@ async function startEc2Instance(label, githubRegistrationToken) {
   const userData = buildUserDataScript(githubRegistrationToken, label);
 
   const params = {
+    ...config.input.ec2VolumeSize && {
+        BlockDeviceMappings: [
+        {
+          DeviceName: "/dev/sda1", 
```

Comment:
You need to also allow the user to specify this. The image I use has the root device name of `/dev/xvda`